### PR TITLE
Updated NV page for new 2.6.7 config for hardened clusters

### LINKED ISSUE #4242: [Rancher2] Update additional steps for neuvector installation when the clusters are hardened
**Request Summary:**
In order for the NeuVector app to work properly in a hardened cluster environment, we need to configure additional options.

**Details:**
RKE1 hardened clusters:
While installing neuvector app in edit options navigate to `Other configuration` >> Enable the check box for `Pod Security Policy` and can leave the other fields empty.
RKE2/k3s hardened clusters:
While installing neuvector app in edit options navigate to `Other configuration` >> Enable the check box for `Pod Security Policy` and for each option `Manager runAsUser ID`, `Scanner runAsUser ID`, `Updater runAsUser ID` add a value greater than 0.

https://github.com/rancher/rancher/issues/37799

### PR DESCRIPTION
**Pull request by jtravee**
Closes #4242

### CODE REVIEW

File: content/rancher/v2.6/en/neuvector-integration/_index.md [hunk 1]
```diff
@@ -118,6 +134,22 @@ kubectl patch deploy neuvector-manager-pod -n cattle-neuvector-system --patch '{
...
+You will need to set additional configuration for your hardened cluster environment as follows:
+
+1. Click **☰ > Cluster Management**.
+1. Go to the cluster that you created and click **Explore**.
+1. In the left navigation bar, click **Apps & Marketplace**.
+1. Install (or upgrade to) NeuVector version `100.0.1+up2.2.2`.
+
+   - Under **Edit Options** > **Other Configuration**, enable **Pod Security Policy** by checking the box. Note that  you must also enter a value greater than `zero` for `Manager runAsUser ID`, `Scanner runAsUser ID`, and `Updater runAsUser ID`:
+
+     ![Enable PSP for RKE1 Hardened Cluster]({{<baseurl>}}/img/rancher/psp-nv-rke2.png)
```
Comment (anupama2501) [inline content/rancher/v2.6/en/neuvector-integration/_index.md:150]: this should be Enable PSP for **RKE2** Hardened Cluster

File: content/rancher/v2.6/en/neuvector-integration/_index.md [hunk 2]
```diff
@@ -118,6 +134,22 @@ kubectl patch deploy neuvector-manager-pod -n cattle-neuvector-system --patch '{
 kubectl patch deploy neuvector-scanner-pod -n cattle-neuvector-system --patch '{"spec":{"template":{"spec":{"securityContext":{"runAsUser": 5400}}}}}'
 kubectl patch cronjob neuvector-updater-pod -n cattle-neuvector-system --patch '{"spec":{"jobTemplate":{"spec":{"template":{"spec":{"securityContext":{"runAsUser": 5400}}}}}}}'
 ```
+<br/>
+
+**_New in v2.6.7_**
```
Comment (anupama2501) [inline content/rancher/v2.6/en/neuvector-integration/_index.md:139]: This is the case for both k3s and RKE2 clusters. We should add k3s too

File: content/rancher/v2.6/en/neuvector-integration/_index.md [hunk 3]
```diff
@@ -118,6 +134,22 @@ kubectl patch deploy neuvector-manager-pod -n cattle-neuvector-system --patch '{
...
+
+You will need to set additional configuration for your hardened cluster environment as follows:
+
+1. Click **☰ > Cluster Management**.
+1. Go to the cluster that you created and click **Explore**.
+1. In the left navigation bar, click **Apps & Marketplace**.
+1. Install (or upgrade to) NeuVector version `100.0.1+up2.2.2`.
+
+   - Under **Edit Options** > **Other Configuration**, enable **Pod Security Policy** by checking the box. Note that  you must also enter a value greater than `zero` for `Manager runAsUser ID`, `Scanner runAsUser ID`, and `Updater runAsUser ID`:
```
Comment (btat) [inline content/rancher/v2.6/en/neuvector-integration/_index.md:148]:
```suggestion
   - Under **Edit Options** > **Other Configuration**, enable **Pod Security Policy** by checking the box. Note that you must also enter a value greater than `zero` for `Manager runAsUser ID`, `Scanner runAsUser ID`, and `Updater runAsUser ID`:
```

**Review by btat — CHANGES_REQUESTED** [review]
I may be missing something, but it looks like the screenshots are identical. If that is the case, then you can just reuse the same image instead.

File: content/rancher/v2.6/en/neuvector-integration/_index.md (hunk 1 above)
Comment (jtravee) [reply content/rancher/v2.6/en/neuvector-integration/_index.md:150]: Thanks for catching this! I updated as both RKE2 and K3s.

File: content/rancher/v2.6/en/neuvector-integration/_index.md (hunk 2 above)
Comment (jtravee) [reply content/rancher/v2.6/en/neuvector-integration/_index.md:139]: Updated to indicate applicability to K3s as well.

**Comment by btat** [comment]
> I may be missing something, but it looks like the screenshots are identical. If that is the case, then you can just reuse the same image instead.

Per discussion, the red box around the elements to select is the difference between the two. Please disregard previous comment.

**Review by btat — APPROVED** [review]

**Review by anupama2501 — APPROVED** [review]
lgtm

**Pull request merged**